app/web/features/search/FilterDialog.tsx

```diff
@@ -198,7 +198,7 @@ export default function FilterDialog({
                 id="last_active_filter"
                 className={classes.marginBottom}
                 value={lastActiveFilter}
-                onChange={(e) => setLastActiveFilter(e.target.value as number)}
+                onChange={(e) => setLastActiveFilter(parseInt(e.target.value as string))}
```

Comment:
why changing num of guest? this is the explanation
![image](https://github.com/user-attachments/assets/1d505ed3-c766-4b98-bba8-e9a65d8ffa48)